src/LuigisBoxBundle/Exceptions/ServiceUnavailable.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Answear\LuigisBoxBundle\Exceptions;
+
+class ServiceUnavailable extends \RuntimeException
```

Comment:
I would prefer to have *Exception suffix in exception classes and/or 'Exception' namespace instead of 'Exceptions'

Reply:
Especially that you have some exceptions already suffixed